### PR TITLE
added basic admin content

### PR DESCRIPTION
**Pull request by EItanya**

### CODE REVIEW

File: gloo/docs/operator_guide/observability.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: Observability
+weight: 3
+---
+
+## Observability
```
Comment (rickducott) [inline gloo/docs/operator_guide/observability.md]: We could call this `Observability (Enterprise)`, and also not sure we need the `## Observability` heading here

File: gloo/docs/operator_guide/usage_statistics.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+title: Usage Statistics
+weight: 3
+---
+
+## Usage Statistics
+
+Gloo will periodically collect usage information from running instances. The details of this collection can be found [here](https://github.com/solo-io/go-checkpoint).
```
Comment (rickducott) [inline gloo/docs/operator_guide/usage_statistics.md]: When does it get checked? I think on Gloo startup and also Glooctl invocation?

File: gloo/docs/operator_guide/observability.md (hunk 1 above)
Comment (EItanya) [reply gloo/docs/operator_guide/observability.md]: It's not really enterprise though, I outline the stuff Open source Gloo ships with

Comment (EItanya) [reply gloo/docs/operator_guide/observability.md]: I'll remove the header though

File: gloo/docs/operator_guide/usage_statistics.md (hunk 2 above)
Comment (EItanya) [reply gloo/docs/operator_guide/usage_statistics.md]: Basically one happens on startup, and then on a given interval after that

Comment (rickducott) [reply gloo/docs/operator_guide/usage_statistics.md]: 👍

**Pull request merged**